src/cmd/cmd.zig

```diff
@@ -1249,7 +1263,12 @@ const AppBase = struct {
             allocator,
             entrypoints,
         );
-        const my_shred_version = echo_data.shred_version orelse 0;
+
+        // zig fmt: off
+        const my_shred_version = config.current.shred_version
+            orelse echo_data.shred_version
+            orelse 0;
+        // zig fmt: on
```

Comment:
I also prefer leading operators. I don't understand the choice they made with zig fmt. `+` at the end of the line especially drives me nuts. But it's an uphill battle. There are a lot of places we'd need to change to converge on this pattern.

Reply:
may be a crazy thought but i wonder if we could just fork zig fmt and make it into our own style 🤔